components/Dashboard/Post/PostComment.tsx

```diff
@@ -163,8 +163,8 @@ const PostComment: React.FC<PostCommentProps> = ({
 
         .body-block {
           display: flex;
-          justify-content: space-between;
-          flex: 1;
+          overflow-wrap: anywhere;
```

Comment:
[This CSS property](https://caniuse.com/#feat=mdn-css_properties_overflow-wrap_anywhere) isn't supported in Safari or IE, would `word-wrap: break-word;` work? (say that 5 times fast 😆 )

Reply:
No I tried that and unfortunately it doesn't work 🙁 Good find re: `overflow-wrap` not being supported though!